src/shared/round.ts

```diff
@@ -0,0 +1,17 @@
+import lodashRound from 'lodash/round';
+
+/**
+ * Replacement for toFixed()
+ *
+ * toFixed falsely rounds the number:
+ * > 1.0005.toFixed(3)
+ * 1.000 // instead of 1.001
+ */
+export function round(value: number, n: number, toFixed = true): number | string {
```

Comment:
suggestion: we probably don't need toFixed option no?

Reply:
thought: should we write a few tests for this?

Reply:
We need it to display a consistent decimal length in the route book, otherwise some rows might display a shorter amount of numbers and make it all misaligned.

Reply:
Too simple for tests? Seems unnecessary imo